parser.y

```diff
@@ -6056,6 +6079,18 @@ ShowTargetFilterable:
 			Tp: ast.ShowProcedureStatus,
 		}
 	}
+|	"PUMP" "STATUS"
+	{
+		$$ = &ast.ShowStmt {
+			Tp: ast.ShowPumpStatus,
+		}
+	}
+|	"DRAINER" "STATUS"
```

Comment:
Do we add a test to parser_test.go ?

Reply:
ok，I will add test in master, and then cherry-pick to release branch

Reply:
No need to push a PR only for this, I just give a suggestion